Gitweb:     http://git.kernel.org/linus/ff5e4b51a397568c6a2901903c35a4bfaaf752a4
Commit:     ff5e4b51a397568c6a2901903c35a4bfaaf752a4
Parent:     aa021baa3295fa6e3f367d80f8955dd5176656eb
Author:     Stefan Schmidt <stefan@datenfreihafen.org>
AuthorDate: Thu Nov 12 09:53:50 2009 +0100
Committer:  Jan Kara <jack@suse.cz>
CommitDate: Thu Nov 12 10:24:12 2009 +0100

    fs/jbd: Export log_start_commit to fix ext3 build.
    
    This fixes:
    ERROR: "log_start_commit" [fs/ext3/ext3.ko] undefined!
    
    Signed-off-by: Stefan Schmidt <stefan@datenfreihafen.org>
---
 fs/jbd/journal.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/fs/jbd/journal.c b/fs/jbd/journal.c
index 49d5cd6..4160afa 100644
--- a/fs/jbd/journal.c
+++ b/fs/jbd/journal.c
@@ -73,6 +73,7 @@ EXPORT_SYMBOL(journal_errno);
 EXPORT_SYMBOL(journal_ack_err);
 EXPORT_SYMBOL(journal_clear_err);
 EXPORT_SYMBOL(log_wait_commit);
+EXPORT_SYMBOL(log_start_commit);
 EXPORT_SYMBOL(journal_start_commit);
 EXPORT_SYMBOL(journal_force_commit_nested);
 EXPORT_SYMBOL(journal_wipe);
